vcd/resource_vcd_vapp_vm.go

```diff
@@ -91,34 +110,42 @@ func resourceVcdVAppVm() *schema.Resource {
 				Type:          schema.TypeString,
 			},
 			"initscript": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "A script to be run only on initial boot",
```

Comment:
```suggestion
				Description: "A script to be run only on initial boot. Will be inherited from template if found in it",
```

Reply:
Is this documented somewhere?

Reply:
https://github.com/terraform-providers/terraform-provider-vcd/pull/347 haven't done that. So see opportunity to add missing detail.

Reply:
Does everyone agree on this change?
@lvirbalas @Didainius ?

Reply:
I have just manually checked these two points:
* In fact, when one does `customization.force=true` it also runs the script so this is not fully correct to say "only on initial boot".
* I have added init script to VM, created a catalog template out of it and spawned new VM with .tf without adding init script and the results is - VM has the script set and it is being executed on first boot (even when my `initscript` parameter is not specified).

Reply:
What about this then:

"Script to run on initial boot or with `customization.force=true` set. Also, inherited from catalog template if found in it"

BTW, when template has it implicitly, do we **really** see the script in this `initscript` field after read?

Reply:
I think API returns it, so we see it.

Reply:
> BTW, when template has it implicitly, do we **really** see the script in this `initscript` field after read?

> If I recall correctly - API returns it, so we see it.

No. The answer is we don't. At least here is what I see in the statefile `"initscript": null,`after creating from template and after additionally trigerring `plan/apply`

Reply:
Thank you. Then modified suggestion:
"Script to run on initial boot or with customization.force=true set"

Reply:
updated

Reply:
> > BTW, when template has it implicitly, do we **really** see the script in this `initscript` field after read?
> 
> > If I recall correctly - API returns it, so we see it.
> 
> No. The answer is we don't. At least here is what I see in the statefile `"initscript": null,`after creating from template and after additionally trigerring `plan/apply`

You would see if you create VM from template which had init script configured in it.